lib/dry/auto_inject/strategies/kwargs.rb

```diff
@@ -47,17 +49,17 @@ def initialize(#{initialize_params})
         end
 
         def define_initialize_with_splat(super_method)
-          super_kwarg_names = super_method.parameters.each_with_object([]) { |(type, name), names|
-            names << name if [:key, :keyreq].include?(type)
-          }
+          super_kwarg_names = super_method.parameters.each_with_object([]) do |(type, name), names|
+            names << name if %i[key keyreq].include?(type)
```

Comment:
I don't think that using `%i` here improves readability. In projects lately this is another Rubocop rule that I disable entirely.